src/elements/content-preview/ContentPreview.js

```diff
@@ -91,6 +93,7 @@ type State = {
     isReloadNotificationVisible: boolean,
     currentFileId?: string, // the currently displayed file id in the collection
     prevFileIdProp?: string, // the previous value of the "fileId" prop. Needed to implement getDerivedStateFromProps
+    isThumbnailsOpen: boolean,
```

Comment:
nit: I think technically this should be `areThumbnailsOpen`. If you want to keep the is pattern maybe `isThumbnailSidebarOpen`?